samples/README.md

```diff
@@ -25,25 +25,32 @@ The tutorials listed below are step-by-step instructions to deploy a fully funct
-|| **State store client - observe key** | Observe a key and receive a notification | :yellow_circle: | [.NET](/dotnet/samples/Services/StateStoreObserveKey) | [Rust](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_services/examples/state_store_client.rs) |
-|| **Leased lock client** | Lock a key in the state store shared between applications | [Go](https://github.com/Azure/iot-operations-sdks/tree/main/go/samples/services/leasedlock) | [.NET](/dotnet/samples/Services/LeasedLockClient) | :yellow_circle: |
-|| **Leader election client** | Leader assignment for highly available applications | :yellow_circle: | [.NET](/dotnet/samples/Services/PassiveReplication) | :yellow_circle: |
+| **Services** | **State store client** | Get, set and delete a key | [Go](/go/samples/services/statestore) | [.NET](/dotnet/samples/Services/StateStoreClient) | [Rust](/rust/azure_iot_operations_services/examples/state_store_client.rs) |
+|| **State store client - observe key** | Observe a key and receive a notification | [Go](/go/samples/services/statestore) | [.NET](/dotnet/samples/Services/StateStoreObserveKey) | [Rust](/rust/azure_iot_operations_services/examples/state_store_client.rs) |
+|| **Leased lock client** | Lock a key in the state store shared between applications | [Go](/go/samples/services/leasedlock) | [.NET](/dotnet/samples/Services/LeasedLockClient) | [Rust](/rust/azure_iot_operations_services/examples/lock_client.rs) |
 || **Schema registry client** | Get and set schemas from the registry | [Go](/go/samples/services/schemaregistry) | [.NET](/dotnet/samples/Services/SchemaRegistryClient) | [Rust](/rust/azure_iot_operations_services/examples/schema_registry_client.rs) |
-|| **Akri client** | Notify Akri services of discovered assets | :yellow_circle: | :yellow_circle: | :yellow_circle: |
+|| **ADR discovery** | Notify Akri services of discovered assets | :yellow_circle: | :yellow_circle: | [Rust](/rust/azure_iot_operations_services/examples/adr_discovery.rs) |
+|| **ADR operations** | Azure Device Registry device asset operations | :yellow_circle: | :yellow_circle: | [Rust](/rust/azure_iot_operations_services/examples/adr_device_asset.rs) |
 ||
-| **Codegen** | **Telemetry & command** | A basic telemetry and command | :yellow_circle: | :yellow_circle: | :yellow_circle: |
-|| **Telemetry + primitive schema** | Telemetry using primitive types such as integers, bool and float | :yellow_circle: | :yellow_circle: | :yellow_circle: |
-|| **Telemetry + complex schema** | Telemetry using complex types such as maps and objects | :yellow_circle: | :yellow_circle: | :yellow_circle: |
-|| **Command variants** | Commands using idempotent and cacheable | :yellow_circle: | :yellow_circle: | :yellow_circle: |
+| **Connector** | **Polling-driven scaffolding** | Template for creating polling-driven connectors | :yellow_circle: | :yellow_circle: | [Rust](/rust/sample_applications/sample_connector_scaffolding) |
```

Comment:
I think dotnet has templates for polling and event driven that should probably be linked here